out/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -2,31 +2,93 @@
     .pb-35 {
         padding-bottom: 35px;
     }
+
+    .color-success {
+      color: green;
+    }
+
+    .color-failure {
+      color: red;
+    }
 </style>
 
 <script type="text/javascript">
+  function addClassToElements(elementIds, className) {
+    elementIds.forEach(function(elementId) {
+      var element = document.getElementById(elementId);
```

Comment:
I would either:
a) pass elements directly (not their IDs) so the DOM doesn't have to be queried all the time
b) use `querySelectorAll` to only query the DOM once per function call, not once for each function call and each ID

Reply:
Thanks, very good point, changed it now